### DISCUSSION TITLE
How to customize table border radius?

### DISCUSSION THREAD

**Discussion by khazifire** [heroui-inc/heroui#405] [Help] [answered]
I'm working with the nextui table component and I can't customize the border radius. i tried doing the following and it still could not work 

`  <Table 
          css={{
            borderRadius:"0",
          }} > ...</Table>`

`  <Table 
          css={{
            br:"0",
          }} > ...</Table>`

**Comment by jrgarciadev** [comment, marked as answer]
Hey @khazifire you should use the `containerCss` prop, here's an example:

https://codesandbox.io/s/sandpack-project-forked-z64pgv
